### DISCUSSION TITLE
resizable_tables

### DISCUSSION THREAD

**Discussion by irshadmb** [javamelody/javamelody#1047] [Q&A] [answered]
I want to know resizable_tables.js has been used, found that it was adopted from 
// **** ADAPTED FROM HTML DEMO AT.... *******
// DEMO = http://bz.var.ru/comp/web/resizable.html
// JS = http://bz.var.ru/comp/web/resizable-tables.js
//  ******* ORIGINAL SCRIPT HEADER *******
// Resizable Table Columns.
//  version: 1.0
//
// (c) 2006, bz

Want to know the Apache License is applicable for the same or we have to get license from Bz

**Comment by evernat** [comment, marked as answer]
In my undestanding, this small javascript code was given as a basic example on the web, to be used on any website you want.
Due credit is there. And nothing more is needed.
So yes, the Apache license is compatible with this javascript.

In case, you don't want to use this javascript, you can override the content of this file, in order to do nothing. For that, see https://github.com/javamelody/javamelody/wiki/UserGuideAdvanced#customizing-styles-icons-and-other-resources-in-the-html-reports

**Reply by irshadmb** [reply]
Thanks @evernat for update.
